lib/model/model.go

```diff
@@ -2939,11 +2935,13 @@ func (m *model) CommitConfiguration(from, to config.Configuration) bool {
 
 	m.pmut.RLock()
 	for _, id := range closeDevices {
+		delete(clusterConfigDevices, id)
 		if conn, ok := m.conn[id]; ok {
 			go conn.Close(errDevicePaused)
```

Comment:
We might be sending multiple closes here for the same connection; I'm going to assume that's OK.

Reply:
Yes, that's a noop then (there's a `sync.Once` in `protocol.Connection`).